rita_extender/src/lib.rs

```diff
@@ -0,0 +1,133 @@
+#[macro_use]
+extern crate log;
+
+#[macro_use]
+extern crate lazy_static;
+
+mod error;
+
+use actix_async::System as AsyncSystem;
+use error::RitaExtenderError;
+use rita_common::dashboard::wifi::ExtenderCheckin;
+use rita_common::dashboard::wifi::WifiInterface;
+
+use std::sync::Arc;
+use std::sync::RwLock;
+use std::thread;
+use std::time::Duration;
+use std::time::Instant;
+
+use rita_common::dashboard::wifi::ExtenderUpdate;
+
+const EXTENDER_CHECKIN_TIMEOUT: Duration = Duration::from_secs(5);
+const EXTENDER_LOOP_SPEED: Duration = Duration::from_secs(30);
+pub const DEFAULT_UPSTREAM_ENDPOINT: &str = "http://[{}]:{}/extender_checkin";
```

Comment:
we need to use the routers ip on the lan which would be 192.168.10.1 by default for ipv6 and is harder to predict for ipv6 since it's based on what we're assigned via dhcpv6